samples/wifi/radio_test/src/nrf_wifi_radio_test_shell.c

```diff
@@ -1005,6 +1005,30 @@ static int nrf_wifi_radio_test_set_rx_capture_length(const struct shell *shell,
 	return 0;
 }
 
+static int nrf_wifi_radio_test_set_rx_capture_timeout(const struct shell *shell, size_t argc,
+						      const char *argv[])
+{
+	char *ptr = NULL;
+	unsigned short int val = 0;
+
+	val = strtoul(argv[1], &ptr, 10);
+
+	if (val >= 600) {
```

Comment:
On line 1433 there is a check that the value is not `0`. Shouldn't that also be checked here?